ui/src/App.tsx

```diff
@@ -128,6 +126,12 @@ function App() {
                               <PlayCircleOutlined />
                             </div>
 
+                            <div className="duration">
+                              <Title level={5}>
+                                {lecture.imprint.substring(12, 20)}
```

Comment:
why subsstring? what is the original value?

Reply:
The original value is for "imprint" is "2021-10-27 - 1:06:01" for example. I only need the duration.

Reply:
Genial. Thanks :D